src/scss/components/_list.scss

```diff
@@ -100,4 +109,97 @@
       padding-right: inherit;
     }
   }
+
+  &__title {
+    color: get-color(rastaGray);
+    display: block;
+    font-size: 12px;
+    font-weight: get-weight(bold);
+    text-align: left;
+    text-transform: uppercase;
+  }
+
+  &--on-column {
+    &[class*='col-'] {
+      display: inline-block;
+      margin-left: 0;
+
+      + & {
+        margin-left: auto;
+        margin-right: 0;
+      }
+
+      .list__item {
+        width: 100%;
+      }
+    }
+  }
+
+  &--social {
+    display: flex;
+    flex-grow: 1;
+
+    .list__item {
+      border-radius: 50%;
+      height: 48px;
+      margin-left: 6%;
+      text-indent: -9999px;
+      width: 48px;
+
+      a {
+        padding: 0;
+        width: 100%;
+      }
+
+      img {
+        display: block;
+        width: 100%;
+      }
+
+      &:first-child {
+        margin-left: 0;
+      }
+    }
+  }
+
+  &--app {
+    display: flex;
+    flex-direction: row;
+    flex-grow: 1;
+
+    .list {
```

Comment:
@felipeorlando the `.list__item` has a `:before` empty with a `margin-left`.

Reply:
I will reset this property on `.list--app` modifier.